packages/create-app/index.js

```diff
@@ -254,14 +254,20 @@ async function init() {
 
   write('package.json', JSON.stringify(pkg, null, 2))
 
-  const pkgManager = /yarn/.test(process.env.npm_execpath) ? 'yarn' : 'npm'
+  const pkgManager = /yarn/.test(process.env.npm_execpath)
+    ? 'yarn'
+    : /npm/.test(process.env.npm_execpath)
+    ? 'npm'
+    : 'pnpm'
```

Comment:
I think we should make the fallout to `npm` instead of `pnpm`, for detecting what pm is used, here is a reference
https://github.com/zkochan/packages/blob/master/which-pm-runs/index.js